typings/response.d.ts

```diff
@@ -0,0 +1,47 @@
+
+export interface DiffSummaryResult {
```

Comment:
I think they all have the word `Summary` in the function name,  so either this one should become `DiffResult` or all others should have `Summary` included... I don't mind which way you prefer to do it though.